### PR TITLE
Add ElasticsearchBundle recipe

### PR DESCRIPTION
**Pull request by saimaz**
| Q             | A
| ------------- | ---
| License       | MIT

This bundle provides extensive integration with the official elasticsearch client library.

### CODE REVIEW

File: ongr/elasticsearch-bundle/5.0/config/packages/ongr_elasticsearch.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Read the documentation: http://docs.ongr.io/ElasticsearchBundle
+ongr_elasticsearch:
+    managers:
+        default:
+            index:
+                hosts:
+                    - 127.0.0.1:9200
+                index_name: my-app-index
+            mappings:
+                - AppBundle
```
Comment (fabpot) [inline ongr/elasticsearch-bundle/5.0/config/packages/ongr_elasticsearch.yaml]: We don"t have `AppBundle` in Flex's project, so that looks wrong to me.

Comment (saimaz) [reply ongr/elasticsearch-bundle/5.0/config/packages/ongr_elasticsearch.yaml]: I could rename it to the `YourBundleWithDocuments` or smth like this. It's an example of mapping documents from the bundle.

Comment (saimaz) [reply ongr/elasticsearch-bundle/5.0/config/packages/ongr_elasticsearch.yaml]: Or maybe `AcmeBundle` ?

Comment (fabpot) [reply ongr/elasticsearch-bundle/5.0/config/packages/ongr_elasticsearch.yaml]: What I mean is that we don't have bundles anymore in a Flex application for project's code, but a simple `App` namespace under `src/`.

**Pull request closed without merging**